OpenXmlPowerTools/PresentationBuilder.cs

```diff
@@ -1715,7 +1715,14 @@ private static void CopyExtendedPart(OpenXmlPart oldContentPart, OpenXmlPart new
                     newPart = ((XmlSignaturePart)newContentPart).AddExtendedPart(oldPart.RelationshipType, oldPart.ContentType, fileInfo.Extension);
 
                 relId = newContentPart.GetIdOfPart(newPart);
-                newPart.FeedData(oldPart.GetStream());
+                using (Stream oldObject = oldPart.GetStream(FileMode.Open, FileAccess.Read))
+                using (Stream newObject = newPart.GetStream(FileMode.Create, FileAccess.ReadWrite))
+                {
+                    int byteCount;
+                    byte[] buffer = new byte[65536];
+                    while ((byteCount = oldObject.Read(buffer, 0, 65536)) != 0)
+                        newObject.Write(buffer, 0, byteCount);
```

Comment:
will it work with `Stream.CopyTo()` ?

Reply:
Not sure, copying is implemented this way in a lot of code paths of `PresentationBuilder`.

Reply:
It works, thx